tools/ci/src/ci.rs

```diff
@@ -63,13 +67,17 @@ impl CI {
             flags |= Flag::KEEP_GOING;
         }
 
+        if self.quiet {
+            flags |= Flag::QUIET;
+        }
+
         match &self.command {
             Some(command) => command.prepare(sh, flags),
             None => {
                 // Note that we are running the subcommands directly rather than using any aliases
                 let mut cmds = vec![];
                 cmds.append(&mut commands::FormatCommand::default().prepare(sh, flags));
-                cmds.append(&mut commands::ClippyCommand::default().prepare(sh, flags));
+                //cmds.append(&mut commands::ClippyCommand::default().prepare(sh, flags));
```

Comment:
why was this commented instead of removed?

Reply:
commented to speed up testing, forgot to uncomment